SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1166,8 +1168,38 @@ void USpatialActorChannel::PostReceiveSpatialUpdate(UObject* TargetObject, const
 	Replicator.CallRepNotifies(false);
 }
 
-void USpatialActorChannel::OnCreateEntityResponse(const Worker_CreateEntityResponseOp& Op)
+void USpatialActorChannel::SendCreateEntityRequest(uint32& OutBytesWritten)
 {
+	UE_LOG(LogActorSystem, Log, TEXT("Sending create entity request for %s with EntityId %lld, HasAuthority: %d"), *Actor->GetName(),
```

Comment:
wrong log- however, could this be in the actor system instead? Rather than having each actor channel separately process all the ops to look for the one it wants?

Reply:
Yup, it's been moved there :)